src/Akeneo/Connectivity/Connection/back/Infrastructure/FeatureFlag/AppDeveloperModeFeature.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Infrastructure\FeatureFlag;
+
+use Akeneo\Platform\Bundle\FeatureFlagBundle\FeatureFlag;
+use Akeneo\Platform\Bundle\FeatureFlagBundle\FeatureFlags;
+
+/**
+ * @copyright 2023 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ * @author    JM Leroux <jean-marie.leroux@akeneo.com>
+ */
+final class AppDeveloperModeFeature implements FeatureFlag
```

Comment:
I'd name it like that to be coherent with others feature flag services. Here the service name is too related with a specific feature
```suggestion
final class OnlySaasDevFeatureFlag implements FeatureFlag
```

Reply:
I took the first example I found :| ...

https://github.com/akeneo/pim-community-dev/blob/fa1e683fdbda3595e5ec4c90bd075bee15f79585/src/Akeneo/Pim/Automation/DataQualityInsights/back/Infrastructure/FeatureFlag/AllCriteriaFeature.php#L21

Reply:
Indeed! So, why not put the class in FeatureFlag directory and just keep the service declaration in connectivity?